src/main/java/nerdhub/textilelib/mixins/MixinWorld.java

```diff
@@ -1,55 +1,18 @@
 package nerdhub.textilelib.mixins;
 
 import nerdhub.textilelib.eventhandlers.EventRegistry;
-import nerdhub.textilelib.events.entity.EntitySpawnedEvent;
 import nerdhub.textilelib.events.tick.WorldTickEvent;
-import net.minecraft.entity.Entity;
-import net.minecraft.entity.player.PlayerEntity;
-import net.minecraft.util.math.MathHelper;
 import net.minecraft.world.World;
 import org.spongepowered.asm.mixin.Mixin;
-import org.spongepowered.asm.mixin.Shadow;
 import org.spongepowered.asm.mixin.injection.At;
 import org.spongepowered.asm.mixin.injection.Inject;
 import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
-import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
 
 import java.util.function.BooleanSupplier;
-import java.util.stream.Stream;
 
 @Mixin(World.class)
 public abstract class MixinWorld {
```

Comment:
Were the entity spawning methods removed from world in general and over to server world?

Reply:
yes